framework/web/XmlResponseFormatter.php

```diff
@@ -77,6 +77,10 @@ public function format($response)
      */
     protected function buildXml($element, $data)
     {
+        if ($data instanceof Arrayable) {
```

Comment:
@SilverFire yeah, you are absolutely right, it will be better. Will send changes in a moment